drivers/timer/wch_systick_ch32v00x.c

```diff
@@ -52,13 +67,15 @@ static int ch32v00x_systick_init(void)
 {
 	IRQ_CONNECT(DT_INST_IRQN(0), 0, ch32v00x_systick_irq, NULL, 0);
 
+	SYSTICK->CTLR = 0;
```

Comment:
Note that the reset state of CTLR is zero. This change is fine, but we generally don't do redundant writes like this. It implies that the reset state was somehow unsuitable.